action.yml

```diff
@@ -1,5 +1,17 @@
 name: "Conventional Commitlint"
 description: "A GitHub Action to check conventional commit message"
+inputs:
+  fail_on_error:
+    description: Whether to fail the workflow if commit messages don't follow conventions.
+    default: 'true'
```

Comment:
query: Wouldn't using `true` (boolean value) work here instead of the string?

Reply:
The default values should be passed as String. Here's a reference [StackOverflow thread](https://stackoverflow.com/questions/76292948/github-action-boolean-input-with-default-value )